### DISCUSSION TITLE
Need help to finish config :)

### DISCUSSION THREAD

**Discussion by joaomribeiro24** [djansen1987/SAJeSolar#73] [General]
Hey there! And thank you so much for this!
I have a SAJ R5-3K-S1 connected by wifi (with that black pen underneath it)
I installed it via HACS and then added the config _(If you have a H1 device, add below Sensors and Resouces…)_

1. I can see the entities and the entity list and even if i try to create a grafana char i can see them on the list. Do you guys have an example of a chart that have production of the panels vs total consumption of the house? If yes what entities should i use on the SAJ?
2. If i want to use the Energy option and Add solar Production, which sensor name should i use?

Thanks!

**Comment by djansen1987** [comment]
Hi, could you share the code you now have in the config( **without** username and password) ?

**Reply by joaomribeiro24** [reply]
I managed to solve it! I needed to wait a few bits that the sensor become available after configuration on yaml file :)
